.github/workflows/check.yml

```diff
@@ -0,0 +1,28 @@
+name: Static Analysis
+
+on:
+  pull_request:
+    branches:
+     - 3.*-devel
+     - master
+  push:
+    branches:
+     - 3.*-devel
+     - master
+  schedule:
+    - cron: 0 0 * * 0
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Latest is v3, but of course v2 works too.

Reply:
Thanks, looks like I was copy-pasting from an old how-to :-)